docker-compose.yml

```diff
@@ -11,8 +11,8 @@ services:
       image: mongo
       restart: always
       # DB Ports not exposed by default.
-      # ports: 
-      #   - 27017:27017
+      ports: 
+        - 27000:27017
```

Comment:
Any reason to use a nonstandard port?

Reply:
This is specific for my local setup and shouldn't be part of PR. Restored default settings.